### PR TITLE
 Integration fixes

### PR DESCRIPTION
**Pull request by jthorton**
## Description
Trying to get the failing integration test to pass based on observations in #218 

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] TODO 1

## Questions
- [ ] Question1

## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by jthorton** [comment]
Finally, non-openeye tests pass!

**Comment by jthorton** [comment]
Seems like the newest version of geometric=1 can fix the integration test without openeye, my guess is something to do with the initial conformer made by rdkit which the new geometric can avoid somehow. Importantly its not a bespokefit issue for once... 
Once #235  is in we should merge this and make a note to update the recipe in the next release.

File: openff/bespokefit/tests/executor/services/qcgenerator/test_worker.py [hunk 1]
```diff
@@ -12,8 +12,8 @@ def test_compute_torsion_drive():
     task = Torsion1DTask(
         smiles="[F][CH2:1][CH2:2][F]",
         central_bond=(1, 2),
-        grid_spacing=180,
-        scan_range=(-180, 180),
+        grid_spacing=15,
+        scan_range=(-165, 180),
```
Comment (Yoshanuikabundi) [inline openff/bespokefit/tests/executor/services/qcgenerator/test_worker.py:16]: The integration test only takes a few minutes even with this change, so this is not a big deal... but could you explain the reasoning behind this change so I can keep it in mind for future tests?

**Review by Yoshanuikabundi — APPROVED** [review]
*Legend*, thanks @jthorton. Please merge at your convenience after #235.

**Pull request merged**